dev-python/xdg/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
```

Comment:
Also, I think I agree with the outcome here (giving new deps to the maintainers) but let's check Soap is happy. I think he is though.

Reply:
Sure, I do it like that for all new packages so it should be good to go